packages/terra-application-links/src/tabs/ApplicationTabs.module.scss

```diff
@@ -42,123 +43,180 @@
     height: 100%;
-    justify-content: center;
-    line-height: var(--terra-application-tabs-line-height, 1.4);
+    line-height: var(--terra-application-links-tabs-tab-line-height, 1.4);
+    margin: 0;
     max-width: 100%;
     outline: none;
     padding: 0;
     position: relative;
+    text-align: center;
     text-decoration: none;
-    text-transform: var(--terra-application-tabs-text-transform, none);
-    transition-duration: var(--terra-application-tabs-transition-duration, 0s);
+    text-transform: var(--terra-application-links-tabs-tab-text-transform, none);
+    transition-duration: var(--terra-application-links-tabs-tab-transition-duration, 0s);
     transition-property: background-size;
-    transition-timing-function: var(--terra-application-tabs-transition-timing, ease);
-    width: var(--terra-application-tab-width, 13rem);
+    transition-timing-function: var(--terra-application-links-tabs-tab-transition-timing-function, ease);
+    width: var(--terra-application-links-tabs-tab-width, 13rem);
 
-    span {
+    &::before {
+      background-color: var(--terra-application-links-tabs-tab-divider-before-background-color, #0079be);
+      background-image: var(--terra-application-links-tabs-tab-divider-before-background-image);
+      border-left: var(--terra-application-links-tabs-tab-divider-before-border-left, 1px solid rgba(0, 0, 0, 0.17));
+      border-right: var(--terra-application-links-tabs-tab-divider-before-border-right, 1px solid rgba(255, 255, 255, 0.17));
+      content: '';
+      display: block;
+      height: 100%;
+      left: 0;
+      overflow: hidden;
+      position: absolute;
+      top: 0;
+      width: var(--terra-application-links-tabs-tab-divider-before-width, 2px);
+    }
+
+    .tab-inner {
+      align-items: center;
+      background-color: var(--terra-application-links-tabs-tab-background-color, transparent);
+      background-position: var(--terra-application-links-tabs-tab-background-position, top left);
+      background-repeat: no-repeat;
+      background-size: var(--terra-application-links-tabs-tab-background-size, 0 5px);
+      box-shadow: var(--terra-application-links-tabs-tab-box-shadow);
+      box-sizing: border-box;
+      display: flex;
+      flex: 0 0 auto;
+      height: 100%;
+      justify-content: center;
+      left: var(--terra-application-links-tabs-tab-position-left, 2px);
+      overflow: hidden;
+      padding-left: var(--terra-application-links-tabs-tab-padding-left, 1.0715rem);
+      padding-right: var(--terra-application-links-tabs-tab-padding-right, 1.0715rem);
+      position: absolute;
+      top: 0;
+      transition-duration: var(--terra-application-links-tabs-tab-transition-duration, 0s);
+      transition-property: background-size;
+      transition-timing-function: var(--terra-application-links-tabs-tab-transition-timing-function, ease);
+      width: calc(100% - var(--terra-application-links-tabs-tab-divider-before-width, 2px));
+    }
```

Comment:
This is a little tricky. This block may not scale very well in the long run. There are three different elements sharing the same name scope. 

```scss
  .tab,
  .tab-menu-display,
  .divider-after-last-tab {
    background-color: transparent; // Not themed yet but would collide with .tab-inner theme variable below
    color: var(--terra-application-links-tabs-tab-color, #fff);

    &::before {
      // This specifies divider but is applied onto .tab, .tab-menu-display, and .divider-after-last-tab
      background-color: var(--terra-application-links-tabs-tab-divider-before-background-color, #0079be);
    }

    .tab-inner {
      // This is applied to tab-inner but does not scope it to tab inner.
      background-color: var(--terra-application-links-tabs-tab-background-color, transparent);
    }
}
```

Reply:
Yeah, I think from a scalability perspective, it may be better to split these selectors and variables up even though they all have the same value in this default theme.

Reply:
It looks like the `&::before` section that specifies divider on its variable names isn't intended to mean only the `.divider-after-last-tab` class but all 3 classes. It generates the borders between each tab in the tab menu.

Reply:
Okay as per offline discussion with @neilpfeiffer, I have undone the selector split.